standalone/runtime/src/lib.rs

```diff
@@ -320,19 +323,6 @@ impl InstanceFilter<Call> for ProxyType {
 				Call::Elections(..) | Call::Treasury(..)
 			),
 			ProxyType::Staking => matches!(c, Call::Staking(..)),
-			ProxyType::StakePoolManager => matches!(
```

Comment:
Should add it back, maybe only for StakePoolv2. This is used to manage stake pool workers.

Reply:
re--added